models/optional_modules/core_web_vitals/scratch/default/snowplow_unified_web_vital_events_this_run.sql

```diff
@@ -69,10 +69,6 @@ with prep as (
 
   -- exclude bot traffic
 
-  {% if var('snowplow__enable_iab', false) %}
-    and not e.iab__spider_or_robot = True
```

Comment:
We don't we go with

`and not e.iab__spider_or_robot` ? Not important ofc, but its redundant

Reply:
Or at least e.iab__spider_or_robot = False, so we skip the not, and we mention what matters ( what we keep ) .

Though fair point is we are creating a macro to exclude so what we dont keep also makes sense to be mentioned